aeson.cabal

```diff
@@ -162,6 +162,9 @@ test-suite tests
     vector,
     quickcheck-instances >=0.3.12
 
+  if !impl(ghc >= 8.0)
+    build-depends: semigroups >= 0.16.1
```

Comment:
why no upper bound? maybe a comment would useful to explain why an upper bound may not be needed